compiler/rustc_interface/src/tests.rs

```diff
@@ -813,6 +813,10 @@ fn test_unstable_options_tracking_hash() {
     tracked!(packed_bundled_libs, true);
     tracked!(panic_abort_tests, true);
     tracked!(panic_in_drop, PanicStrategy::Abort);
+    tracked!(
+        patchable_function_entry,
+        PatchableFunctionEntry::from_total_and_prefix_nops(10, 5).expect("total >= prefix")
```

Comment:
The message in `expect` needs to be the description of the condition that *wasn't* met.

```suggestion
        PatchableFunctionEntry::from_total_and_prefix_nops(10, 5).expect("total < prefix")
```

Reply:
Don't know if I understood you correctly, but to get a value the condition 'total >=prefix' must be true. The panic would happen if that condition is false. So I'd think it would be correct as written?

Reply:
What will happen if `total < prefix` is that at runtime we'll get a panic that looks like the following:

```
thread 'main' panicked at compiler/rustc_interface/src/tests.rs:818:8:
total >= prefix
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=7e6fd3fe91aec1a58fdd5b198388ec23

Instead we should provide additional context beyond that. Having said that, this is a nitpick to make it easier for people to understand that happened in this test, it's not a deal breaker.

Reply:
```suggestion
        PatchableFunctionEntry::from_total_and_prefix_nops(10, 5).expect("total must be greater than prefix")
```

Reply:
maybe?

Reply:
or just unwrap, this is just a test, really, and the condition is trivially obviously true.